src/ftw_cli/download.py

```diff
@@ -14,22 +14,11 @@ def ftw():
     pass
 
 @click.command(help="Download the FTW dataset.")
-@click.option('--clean_download', is_flag=True, help="If set, the script will delete the root folder before downloading.")
-@click.option('--root_folder', type=str, default="./data", help="Root folder where the files will be downloaded. Defaults to './data'.")
-@click.option('--countries', type=str, default="all", help="Comma-separated list of countries to download. If 'all' is passed, downloads all available countries.")
-def download(clean_download, root_folder, countries):
-    # Use the provided root_folder or prompt the user for it
-    if root_folder is None:
-        root_folder = click.prompt(
-            "Please enter the root folder for downloaded files",
-            default=os.path.abspath('./data'),
-            show_default=True
-        )
```

Comment:
This actually never happens because a default value is provided...

Reply:
Oh yeah, I think this got lost in the mix between me and Aninda. I sorta liked the idea of prompting people instead of just installing to a /data folder from wherever they happened to be, but also making it easy for them to just hit 'yes' if they were fine with it. I'd slightly prefer to bring it back, but curious on what you think Matthias. I suppose the 'pattern' in fiboa CLI is generally just require -o. This one is a bit different though since it's a folder, and it's a place where you'll get lots of data...

Reply:
Yeah, I usually think default with an option to change it is better here. Then it can run more easily in tests and GH Actions, too...